services/navigation/builtin/builtin.go

```diff
@@ -126,9 +127,9 @@ func (conf *Config) Validate(path string) ([]string, error) {
 		deps = append(deps, resource.NewName(motion.API, resource.DefaultServiceName).String())
 	}
 
-	// Add vision service dependencies
-	for _, v := range conf.VisionServices {
-		deps = append(deps, resource.NewName(vision.API, v).String())
+	for _, obstacleDetectorPair := range conf.ObstacleDetectors {
```

Comment:
[minor] What limits are we putting on what makes a valid `obstacle_detector`? My first thought is that a vision service is required to be present, I'm a bit more unclear of if a camera is necessary as well. 

I know explore mode will require a camera, but I'm not sure about all the other edge cases of motion

Reply:
i think it is fair to assume that an `obstacle_detector` is valid iff it has a vision service **and** camera.
I will verify tmr with the vision team

Reply:
Confirmed that to make a valid `obstacle_detector` we need some type of camera (maybe a lidar). Otherwise we have no sensor to give us input.